canonical-web-and-design/vanilla-framework
# Spacing adjustment

## Done

This is a work in progress branch that aims to:
- make text spacing slightly simplified
- make shallow strip match the standard bottom spacing on the majority of elements. This means go from 2rem to 1.5rem; 
- make headings slightly more spaced from generic divs like images etc, which was requested in https://github.com/canonical-web-and-design/vanilla-framework/issues/2420
- makes spacing under button match that under inputs and many other elements

## QA

- Pull code
- Run `./run serve --watch`
- Open examples/templates/typographic-spacing/ and scroll to the bottom
- Inspect the examples showing headings and paragraphs with different parents matching those in the same parents:
![image](https://user-images.githubusercontent.com/2741678/62557108-3ebd0880-b86e-11e9-8193-cd7f4f42e7eb.png)
- Verify the image sits further from the heading:
![image](https://user-images.githubusercontent.com/2741678/62557139-4c728e00-b86e-11e9-82f0-c112b2a5a30e.png)

- QA percy for visual changes
